common/utils.py

```diff
@@ -30,9 +32,8 @@ def url_is_image(url: str) -> bool:
     return False
 
 
-def reddit_post_is_relevant(
```

Comment:
sorry for the formatting changes here. I have line lenght set to 120 in text editor